src/PIL/EpsImagePlugin.py

```diff
@@ -324,32 +325,51 @@ def _read_comment(s: str) -> bool:
                     int(value) for value in image_data_values[:4]
                 )
 
+                imagedata_size = (columns, rows)
+
                 if bit_depth == 1:
                     self._mode = "1"
                 elif bit_depth == 8:
                     try:
                         self._mode = self.mode_map[mode_id]
                     except ValueError:
-                        break
+                        pass
                 else:
-                    break
-
-                self._size = columns, rows
-                return
+                    pass
```

Comment:
Couldn't
```python
else:
    pass
```
just be removed?

Reply:
Yes. I'm used to using a linter that requires an `else` for every `if`/`elif`, but that's not the case for this project.